include/rocksdb/rate_limiter.h

```diff
@@ -26,9 +26,9 @@ class RateLimiter {
   };
 
   enum class Mode {
-    kReadsOnly,
-    kWritesOnly,
-    kAllIo,
+    kReadsOnly = 0,
```

Comment:
no functional change here, but looking at other enums that have been exposed via the c api, they explicitly define the enum values so updated here to match that pattern